kinesis.go

```diff
@@ -76,7 +77,7 @@ func makeParams(action string) map[string]string {
 // RequestArgs store params for request
 type RequestArgs struct {
 	params  map[string]interface{}
-	Records []PutRecordsRecord
+	Records []RecordToPut
```

Comment:
Yeah `Record` and therefore `Records []Record` would make a ton of sense and would probably work fine… I didn't contemplate such a change because in the interest of getting my contribution accepted I tried to keep the scope of my changes as narrow as possible. Would you like me to add that change to this branch?